modules/nf-core/seq2hla/meta.yml

```diff
@@ -0,0 +1,189 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "seq2hla"
+description: Precision HLA typing and expression from RNA-seq data using seq2HLA
+keywords:
+  - hla
+  - typing
+  - rna-seq
+  - genomics
+  - immunogenetics
+tools:
+  - "seq2hla":
+      description: "Precision HLA typing and expression from next-generation RNA sequencing data"
+      homepage: "https://github.com/TRON-Bioinformatics/seq2HLA"
+      documentation: "https://github.com/TRON-Bioinformatics/seq2HLA#readme"
+      tool_dev_url: "https://github.com/TRON-Bioinformatics/seq2HLA"
+      doi: "10.1186/s13073-015-0240-5"
+      licence: ['MIT']
+      identifier: biotools:seq2HLA
+
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. `[ id:'sample1', single_end:false ]`
+    - reads:
+        type: file
+        description: Paired-end FASTQ files for RNA-seq data
+        pattern: "*.{fastq.gz,fq.gz}"
+        ontologies:
+          - edam: http://edamontology.org/format_1930 # FASTQ
+
+output:
+  class1_genotype_2d:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*ClassI-class.HLAgenotype2digits":
+          type: file
+          description: HLA Class I 2-digit genotype results
+          pattern: "*ClassI-class.HLAgenotype2digits"
+          ontologies: []
```

Comment:
I guess this is just a text file? Or is it a TSV file?

Reply:
tsv-ish file, but it doesn't end on tsv and the header starts with #

```
#Locus	Allele 1	Confidence	Allele 2	Confidence
A	no	NA	hoz("A*24")	NA
B	no	NA	hoz("B*35")	NA
C	no	NA	hoz("C*17")	NA
```